scripts/check_edge_docs.py

```
@@ -0,0 +1,164 @@
+        title = raw.strip('"\'')
+        titles.append(title)
+    titles.sort(key=str.casefold)
+    return titles
+
+
+def extract_edge_keys_from_code(index_path: Path) -> List[str]:
+    text = read_file_text(index_path)
+    # Find the EdgeInfoComponents object literal
+    start = text.find("const EdgeInfoComponents = {")
+    if start == -1:
+        raise RuntimeError("Could not find 'const EdgeInfoComponents = {' in index.tsx")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)